pkg/controllers/disruption/controller.go

```diff
@@ -230,9 +230,21 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command,
 	// We have the new NodeClaims created at the API server so mark the old NodeClaims for deletion
 	c.cluster.MarkForDeletion(providerIDs...)
 
+	// Set the status of the nodeclaims to reflect that they are disruption candidates
+	err = multierr.Combine(lo.Map(cmd.candidates, func(candidate *Candidate, _ int) error {
+		candidate.NodeClaim.StatusConditions().SetTrueWithReason(v1.ConditionTypeDisruptionCandidate, v1.ConditionTypeDisruptionCandidate, m.EvictionReason(candidate.NodeClaim))
+		return c.kubeClient.Status().Update(ctx, candidate.NodeClaim)
+	})...)
+	if err != nil {
+		return multierr.Append(fmt.Errorf("updating nodeclaim status: %w", err), state.RequireNoScheduleTaint(ctx, c.kubeClient, false, stateNodes...))
+	}
+
 	if err := c.queue.Add(orchestration.NewCommand(nodeClaimNames,
 		lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode { return c.StateNode }), commandID, m.Reason(), m.ConsolidationType())); err != nil {
 		c.cluster.UnmarkForDeletion(providerIDs...)
+		err = multierr.Combine(err, multierr.Combine(lo.Map(cmd.candidates, func(candidate *Candidate, _ int) error {
+			return multierr.Append(candidate.NodeClaim.StatusConditions().Clear(v1.ConditionTypeDisruptionCandidate), c.kubeClient.Status().Update(ctx, candidate.NodeClaim))
```

Comment:
We try to do this at the top of the disruption loop now. You should be able to clear the condition for any nodeclaim that doesn't exist in the orchestration queue